crates/modelardb_server/src/storage/data_transfer.rs

```diff
@@ -439,22 +474,20 @@ mod tests {
                 .table_size_in_bytes
                 .get(test::MODEL_TABLE_NAME)
                 .unwrap(),
-            FILE_SIZE * 2
+            MODEL_TABLE_FILE_SIZE * 2
         );
     }
 
     #[tokio::test]
     async fn test_set_transfer_batch_size_in_bytes_to_none() {
-        let temp_dir = tempfile::tempdir().unwrap();
-        let local_data_folder = create_local_data_folder(&temp_dir).await;
-
-        create_delta_table_and_segments(local_data_folder.clone(), 2).await;
+        let (_temp_dir, local_data_folder) = create_local_data_folder_with_tables().await;
 
+        write_batches_to_tables(&local_data_folder, 2).await;
         let (_, mut data_transfer) = create_data_transfer_component(local_data_folder).await;
 
         assert_eq!(
             data_transfer.transfer_batch_size_in_bytes,
-            Some(FILE_SIZE * 3 - 1)
+            Some(MODEL_TABLE_FILE_SIZE * 3 - 1)
```

Comment:
Added a comment to where we create the test `DataTransfer` component since this check is just verifying that the transfer batch size is the originally set value.